### PR TITLE
fix #116751: support for ampeg_vel2* opcodes

### PR DESCRIPTION
**Pull request by hpfmn**

### CODE REVIEW

File: zerberus/sfz.cpp [hunk 1]
```diff
@@ -266,63 +285,76 @@ static void readInt(const QString& data, int* val)
 
 void SfzRegion::readOp(const QString& b, const QString& data)
       {
-      int i = data.toInt();
+      QStringList splitData = data.split(" ");
```
Comment (lasconic) [inline zerberus/sfz.cpp]: Could you add a comment here.
*No space in opcode value except for sample*
Or something like that.

Comment (hpfmn) [inline zerberus/sfz.cpp]: done

File: zerberus/voice.cpp [hunk 2]
```diff
@@ -145,22 +147,24 @@ void Voice::start(Channel* c, int key, int v, const Zone* z)
 
       currentEnvelope = V1Envelopes::DELAY;
 
+      float velPercent = _velocity / 127.0;
```
Comment (lasconic) [inline zerberus/voice.cpp]: Did you read it somewhere that the impact should be "linear"?

Comment (hpfmn) [inline zerberus/voice.cpp]: Simon Cann "Cakewalk Synthesizers": p. 407

Comment (lasconic) [inline zerberus/voice.cpp]: Indeed, thanks.

**Pull request merged**